src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin-top: 20px;
+  font-family: "Avenir-Book", sans-serif;
+}
+
+.input {
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  font-family: "Avenir-Book", sans-serif;
```

Comment:
you can use inherit here
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit